### PR TITLE
Add commands!

### PR DESCRIPTION
**Pull request by rpxs**
This pull request adds the following features: 

- [x] Gats Servers Command
- [x] DaBaby Command

I swear these were both requested by the community 😃 😃

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,12 +15,16 @@
     "discord.js": "^12.5.1",
     "electron": "^11.1.1",
     "express": "^4.17.1",
+    "fetch": "^1.1.0",
```
Comment (Dorfnox) [inline package.json:18]: I would prefer to stick to either axios or fetch if we are going to be using one or the other. Can you switch your code from using fetch to using axios?  Here's some differences between the two: https://stackoverflow.com/questions/40844297/what-is-difference-between-axios-and-fetch

File: src/data-layer/TwitchHelix.js [hunk 2]
```diff
@@ -7,8 +7,8 @@ const { address: publicIPAddress, port } = publicIP;
 
 class TwitchHelix {
   static appAccess = {
-    accessToken: null,
-    expiresAt: null, // Time in seconds
+    accessToken: "poggers",
```
Comment (Dorfnox) [inline src/data-layer/TwitchHelix.js]: What is this?

File: src/data-layer/TwitchUserData.js [hunk 3]
```diff
@@ -29,23 +29,24 @@ class TwitchUserData extends ServerCacheManager {
     return this.get(
       _id,
       TwitchUserData.getDefaultTwitchUserData(TwitchUserAPIData)
-    ).then((TwitchUserDBData) => {
```
Comment (Dorfnox) [inline src/data-layer/TwitchUserData.js:32]: As far as I can tell, this is just a slight formatting change. In the future, try to keep formatting changes in separate pull requests... otherwise, pull requests can become large and a burden to sift through for other developers.

File: src/data-layer/WaffleMongo.js [hunk 4]
```diff
@@ -19,8 +19,8 @@ class WaffleMongo {
     if (client) {
       return Promise.resolve(client);
     }
-    const mongDBUrl = `mongodb+srv://${mongoDBUsername}:${mongoDBPassword}@${mongoDBClusterNetAddress}/test?retryWrites=true&w=majority`;
-    return MongoClient.connect(mongDBUrl, { useUnifiedTopology: true })
+    const mongoDBUrl = `mongodb+srv://${mongoDBUsername}:${mongoDBPassword}@${mongoDBClusterNetAddress}/${mongoDatabaseName}?retryWrites=true&w=majority`;
```
Comment (Dorfnox) [inline src/data-layer/WaffleMongo.js:22]: :D

File: src/gats/GatsConstants.json [hunk 5]
```diff
@@ -1,7 +1,8 @@
...
-    "gatsChangeLogURL": "https://stats.gats.io/changelog",
-    "vipLogoUrl": "https://gats.io/img/vip.png"
-}
\ No newline at end of file
+  "gatsUrl": "https://gats.io/",
+  "statsUrl": "https://stats.gats.io/",
+  "gatsLogoUrl": "https://stats.gats.io/img/gats_logo.png",
+  "gatsChangeLogURL": "https://stats.gats.io/changelog",
+  "vipLogoUrl": "https://gats.io/img/vip.png",
+  "serverUrl": "https://io-8.com/find-instances"
```
Comment (Dorfnox) [inline src/gats/GatsConstants.json:7]: What is io-8?

File: src/gats/GatsRequests.js [hunk 6]
```diff
@@ -133,6 +133,32 @@ class GatsRequests {
       });
   }
 
+  static requestGatsServers() {
```
Comment (Dorfnox) [inline src/gats/GatsRequests.js:136]: As I suggested before, I would prefer to stick to axios for now.

File: src/gats/GatsRequests.js [hunk 7]
```diff
@@ -223,7 +249,7 @@ class GatsRequests {
           `#pageContainer > div:nth-child(${nthChild}) > div:nth-child(2) > div > table > tbody > tr`;
 
         // Collect vip details
-        var vip = null;
+        let vip = null;
```
Comment (Dorfnox) [inline src/gats/GatsRequests.js:252]: Good call - but again, try to keep cleanup code to seperate pull requests.

File: src/message/WaffleResponse.js [hunk 8]
```diff
@@ -1,110 +1,131 @@
-const { getSafe, randomWaffleColor } = require('../util/WaffleUtil');
```
Comment (Dorfnox) [inline src/message/WaffleResponse.js:1]: This entire code seems to have been prettified. Again, plz leave formatting changes for separate pull requests so that your main code changes can be the focus.

File: package.json (hunk 1 above)
Comment (rpxs) [reply package.json:18]: Sounds good!

File: src/message/WaffleResponse.js (hunk 8 above)
Comment (rpxs) [reply src/message/WaffleResponse.js:1]: Alright, sounds good!

File: src/data-layer/TwitchHelix.js (hunk 2 above)
Comment (rpxs) [reply src/data-layer/TwitchHelix.js]: I've added placeholder data to the twitchhelix file, because it doesnt seem to run locally without it. Node would throw some errors because accessToken would be undefined.

**Comment by rpxs** [comment]
Added some clarification!

**Review by Dorfnox — APPROVED** [review]
Awesome!
